python/ctsm/modify_fsurdat/modify_fsurdat.py

```diff
@@ -0,0 +1,347 @@
+"""
+Run this code by using the following wrapper script:
+/tools/modify_fsurdat/fsurdat_modifier
+
+The wrapper script includes a full description and instructions.
+"""
+
+import os
+
+import numpy as np
+import xarray as xr
+
+from ctsm.utils import abort, get_git_sha, update_metadata, lon_range_0_to_360
+
+class ModifyFsurdat:
+    """
+    A case to encapsulate function that modifies fsurdat file.
+
+    ...
+
+    Attributes
+    ----------
+
+    Methods
+    -------
+    write_output:
+        Write out the modified fsurdat (surface dataset)
+
+    TODO: Complete list of methods here?
+    """
+
+    def __init__(self, fsurdat_in):
+
+        print("Open file: " + fsurdat_in)
+        self.file = xr.open_dataset(fsurdat_in)
```

Comment:
Renamed `self.file` to `self._file` and back to `self.file` because in the refactor from set_in_rectangle to set_idealized `self.file` is not private.